seed-job/Jenkinsfile

```diff
@@ -0,0 +1,5 @@
+monorepoSeedPipeline {
+  platform = ['ubuntu']
+  organizationName = 'Symbol'
+  gitHubId = 'Symbol-Github-app'
```

Comment:
check if needed - see #46 git checkout removal

Reply:
Only needed for private repo or if we still plan to automatic release where the Jenkins needs to update the Github